src/modules/token/fungible/components/tokenAmount/index.js

```diff
@@ -39,7 +39,7 @@ const TokenAmount = ({
   if (showInt) value = getInt(value);
   else if (showRounded) value = trim(value);
   return (
-    <Wrapper className={className}>
+    <Wrapper {...(className && { className })}>
```

Comment:
@oskarleonard what we gain from this condition? className are fine with null or undefiend

Reply:
If we pass in React.Fragment as a wrapper react will complain if classname is a prop even if it is undefined. So this prevents that warning from happening.